src/java.base/share/classes/java/io/Console.java

```diff
@@ -406,7 +406,8 @@ private static Console instantiateConsole(boolean istty) {
             /*
              * The JdkConsole provider used for Console instantiation can be specified
              * with the system property "jdk.console", whose value designates the module
-             * name of the implementation, and which defaults to "java.base". If no
+             * name of the implementation, and which defaults to the value
```

Comment:
Might read slightly better:
```suggestion
             * name of the implementation, and which defaults to the value of
```